include/oneapi/dnnl/dnnl_graph.hpp

```diff
@@ -819,6 +819,7 @@ class op : public op_handle {
         Reciprocal = dnnl_graph_op_reciprocal,
         ReduceL1 = dnnl_graph_op_reduce_l1,
         ReduceL2 = dnnl_graph_op_reduce_l2,
+        ReduceAMax = dnnl_graph_op_reduce_amax,
```

Comment:
Thanks for your PR for Graph API. As you may know, not all graph OPs are directly mapped to primitive. Such as amax, it can be integrated as Graph API OPs. [Abs](https://oneapi-src.github.io/oneDNN/dev_guide_op_abs.html), and [ReduceMax](https://oneapi-src.github.io/oneDNN/dev_guide_op_reducemax.html)  amax(x) = max(abs(x)). then in Graph API backend, it can be fused to amax then mapped to primitive. May I know if there're any requirement or background for Graph API to add this new OP?

Reply:
I am not sure if there is any requirement; I tried to add amax because cublaslt and hipblaslt already support the operation  

Thanks